doc/source/webapp_ui/graph_view/electron.rst

```diff
@@ -0,0 +1,30 @@
+===================
+Electron Sidebar
+===================
+
+.. image:: ../images/electron_sidebar.gif
+   :align: center
+   :width: 5000px
+
+Electron sidebar gives a node level detail for a particular workflow dispatch.
+The sidebar will be shown on the right-hand side when one of the nodes is clicked. This sidebar contains the following attributes:
+
```

Comment:
In the attributes bulleted lists, can you tab the descriptions of each attribute? It would format like:

- Started - Ended
     Shows the local time when a dispatch started and when a dispatch ended.

Also, under the last attribute in lattice.rst and electron.rst let's get rid of the spaces after the parenthesis. So "( python format )" --> "(python format)"

Otherwise looks good!